src/fields/EstimatedCall.ts

```diff
@@ -29,7 +29,7 @@ export interface EstimatedCall {
     actualDepartureTime?: string // Only available AFTER departure has taken place
     aimedArrivalTime: string
     aimedDepartureTime: string
-    bookingArrangements?: BookingArrangement
+    bookingArrangements?: BookingArrangement // @deprecated
```

Comment:
Eg trur du må bruke denne syntaksen for at TypeScript skal registrere det riktig som TSDoc:
```
/** @deprecated Use bookingArrangements on Leg instead. */
bookingArrangements?: BookingArrangement 
```

Også lurt å legge på beskrivelse for å hjelpe brukarane på veg.